server/src/hvy/treeBuilder.ts

```diff
@@ -900,39 +798,36 @@ export class TreeBuilder
         {
             var funcNode: FunctionCallNode = new FunctionCallNode();
 
-            if (codeLevel[1][0] == "ns")
-            {
+            if (codeLevel[1][0] == "ns" && Array.isArray(codeLevel[1]) && Array.isArray(codeLevel[1][1])) {
```

Comment:
The `Array.isArray(codeLevel[1])` check should be before `codeLevel[1][0] == "ns"`